recipes/azure-mgmt-advisor/LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright (c) Microsoft Corporation.
```

Comment:
Issue filed: https://github.com/Azure/azure-sdk-for-python/issues/40981
License taken from: https://github.com/Azure/azure-sdk-for-python/blob/d7d04c00f8837a87692aa086dd7c4a26d508344b/sdk/advisor/azure-mgmt-advisor/LICENSE